queries/windows_filespace.kusto

```diff
@@ -30,6 +30,6 @@ InsightsMetrics
 | extend script_name = "n/a"
 | extend script_version = "n/a"
 | extend value = round(Val, 2)
-| extend additional_information = strcat("Only ", (value), " % Diskspace free. Managed by ",(tags_managedby_s),".")
+| extend additional_information = strcat("Only ", (value), " % Diskspace free. Managed by ", (tags_managedby_s), ".")
 | extend affected_entity = Mountpoint
-| project TimeGenerated = now(), _ResourceId, state, affected_object, monitor_package, monitor_name, monitor_description, script_name, script_version, threshold, value, affected_entity, additional_information
\ No newline at end of file
+| project _ResourceId, RawData = strcat("TimeGenerated:", now(), ";_ResourceId:", _ResourceId, ";state:", state, ";affected_object:", affected_object, ";monitor_package:", monitor_package, ";monitor_name:", monitor_name, ";monitor_description:", monitor_description, ";script_name:", script_name, ";script_version:", script_version, ";threshold:", threshold, ";value:", value, ";affected_entity:", affected_entity, ";additional_information:", additional_information)
```

Comment:
I don't think we should "define" ou own format. Use json instead.

Reply:
Good suggestion. Will not be done in this PR sadly